cmd/restic/cmd_forget.go

```diff
@@ -64,6 +69,11 @@ func init() {
 	f.IntVarP(&forgetOptions.Monthly, "keep-monthly", "m", 0, "keep the last `n` monthly snapshots")
 	f.IntVarP(&forgetOptions.Yearly, "keep-yearly", "y", 0, "keep the last `n` yearly snapshots")
 	f.VarP(&forgetOptions.Within, "keep-within", "", "keep snapshots that are newer than `duration` (eg. 1y5m7d2h) relative to the latest snapshot")
+	f.VarP(&forgetOptions.HourlyWithin, "keep-hourly-within", "", "keep hourly snapshots that are newer than `duration` (eg. 1y5m7d2h) relative to the latest snapshot")
```

Comment:
After discussing with @rawtaz we'd prefer to name the options such that they are prefixed with `keep-within` to group the new options together in the help output. That is `--keep-hourly-within` would become `--keep-within-hourly`, etc.

Reply:
OK, that makes sense. I'll update that, and also the variable names in the code for consistency. It will be a couple of weeks since I'm traveling.

Reply:
Swapping the order should be done now.